r/R/table.R

```diff
@@ -146,21 +146,15 @@ Table <- R6Class("Table", inherit = ArrowObject,
       if (is.integer(i)) {
         i <- Array$create(i)
       }
-      if (inherits(i, "ChunkedArray")) {
-        return(shared_ptr(Table, Table__TakeChunked(self, i)))
-      }
-      assert_is(i, "Array")
-      shared_ptr(Table, Table__Take(self, i))
+      # Invalid: Tried executing function with non-value type: Table
```

Comment:
Another comment to remove?

Reply:
Yes probably -- I got confused about what the comment meant. Do you think https://issues.apache.org/jira/browse/ARROW-9001 is feasible?